cmd/preflight/cmd/check_operator.go

```diff
@@ -11,28 +11,29 @@ import (
 	"github.com/redhat-openshift-ecosystem/openshift-preflight/internal/formatters"
 	"github.com/redhat-openshift-ecosystem/openshift-preflight/internal/lib"
 	"github.com/redhat-openshift-ecosystem/openshift-preflight/internal/runtime"
-	"github.com/redhat-openshift-ecosystem/openshift-preflight/internal/viper"
 	"github.com/redhat-openshift-ecosystem/openshift-preflight/operator"
 	"github.com/redhat-openshift-ecosystem/openshift-preflight/version"
 
 	"github.com/go-logr/logr"
 	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
 )
 
-func checkOperatorCmd(runpreflight runPreflight) *cobra.Command {
+func checkOperatorCmd(runpreflight runPreflight, viper *viper.Viper) *cobra.Command {
 	checkOperatorCmd := &cobra.Command{
 		Use:   "operator",
 		Short: "Run checks for an Operator",
 		Long:  `This command will run the Certification checks for an Operator bundle image. `,
-		Args:  checkOperatorPositionalArgs,
+		Args: func(cmd *cobra.Command, args []string) error {
+			return checkOperatorPositionalArgs(args, viper)
+		},
```

Comment:
Should we extract this closure into its own function? As in, to reduce in-line function definitions where not needed?

Reply:
Either way. Since it's a single line I'm not super worried about this one.